spatialconnect/src/main/java/com/boundlessgeo/spatialconnect/services/SCBackendService.java

```diff
@@ -52,7 +54,8 @@
     private static Context context;
     private static MqttHandler mqttHandler;
     private static HttpHandler httpHandler;
-
+    private static BehaviorSubject<SCNotification> notifyTopic = BehaviorSubject.create();
+    public static ConnectableObservable<SCNotification> notifications = notifyTopic.publish();
```

Comment:
@tetriscode What do you think about handling notifications with the ConnectableObservable?